src/commands/sbom/upload.ts

```diff
@@ -70,6 +71,14 @@ export class UploadSbomCommand extends Command {
       return 1
     }
 
+    this.config.env = this.config.env || this.env
```

Comment:
```suggestion
    this.config.env = this.env || this.config.env
```

I'd rather a passed in value override an env var. This behavior is consistent with the service on line 66 above